src/StringValidatorFiller.php

```diff
@@ -34,13 +38,74 @@ class StringValidatorFiller implements IValidatorFiller
      */
     public function fill(BaseValidator $validator)
     {
-        $minLength = $validator->minLength;
-        $maxLength = $validator->maxLength ?? $validator->minLength + 1;
+        if ($validator->format !== null && $validator->format === 'date-time') {
+            $dateMin = (new \DateTime())
+                ->modify('-1 months')->getTimestamp();
+
+            if ($validator->formatMinimum !== null) {
+                $dateMin = (new \DateTime(
+                    $validator->formatMinimum
+                ))->getTimestamp();
+            }
+
+            $dateMax = (new \DateTime())
+                ->modify('+1 months')->getTimestamp();
+            if ($validator->formatMaximum !== null) {
+                $dateMax = (new \DateTime(
+                    $validator->formatMaximum
+                ))->getTimestamp();
+            }
+
+            $randomEpoch = mt_rand($dateMin, $dateMax);
+
+            $randomString = (new \DateTime(
```

Comment:
can you do early exit instead of branching with else?

Reply:
Done.